zerver/lib/bulk_create.py

```diff
@@ -146,6 +147,18 @@ def bulk_create_users(
             )
 
     UserGroupMembership.objects.bulk_create(group_memberships_to_create)
+    now = timezone_now()
+    RealmAuditLog.objects.bulk_create(
+        RealmAuditLog(
+            realm=membership.user_group.realm,
```

Comment:
This function already has realm available. Fetching it from user_group will involve an extra query I guess.